### ISSUE TITLE
enable support for `java.sql.Timestamp`

### ISSUE THREAD

**Issue by mamdouhweb** [outworkers/phantom#823]
Following up on our conversation over [Gitter](https://gitter.im/outworkers/phantom?at=5a8ab6e98f1c77ef3a27e794).

_Use case:_
Error occurs while using Timestamp as a DataType for case class fields

_Stacktrace:_
```scala
scala.NotImplementedError: an implementation is missing
        at scala.Predef$.$qmark$qmark$qmark(Predef.scala:230)
        at mapper.gender.YearlyGenderDistributionTable$anon$macro$16$1.fromRow(GenderDistributionMapper.scala:11)
        at mapper.gender.YearlyGenderDistributionTable$anon$macro$16$1.fromRow(GenderDistributionMapper.scala:11)
        at com.outworkers.phantom.CassandraTable.fromRow(CassandraTable.scala:63)
        at com.outworkers.phantom.SelectTable$$anonfun$select$1.apply(SelectTable.scala:27)
        at com.outworkers.phantom.SelectTable$$anonfun$select$1.apply(SelectTable.scala:27)
        at com.outworkers.phantom.builder.query.SelectQuery.fromRow(SelectQuery.scala:59)
``` 

_Example:_
```scala
abstract class YearlyGenderDistributionTable
    extends Table[YearlyGenderDistributionTable, CompanyWideGenderDistributionYearly] {
  object enterprise_id           extends StringColumn with PartitionKey
  object year                    extends DateTimeColumn with ClusteringOrder with Descending
  object hired_female_count      extends BigIntColumn
  object terminated_female_count extends BigIntColumn
  object hired_male_count        extends BigIntColumn
  object terminated_male_count   extends BigIntColumn
  object total_female_count      extends BigIntColumn
  object total_male_count        extends BigIntColumn
}

CREATE TABLE IF NOT EXISTS dashboard.gender_representation_yearly (
    enterprise_id text,
    year timestamp,
    hired_female_count bigint,
    terminated_female_count bigint,
    hired_male_count bigint,
    terminated_male_count bigint,
    total_female_count bigint,
    total_male_count bigint,

    PRIMARY KEY ((enterprise_id), year)
) WITH CLUSTERING ORDER BY (year DESC);
```

**Comment by alexflav23** [comment]
Hi @mamdouhweb I think this is a more interesting problem where something may be incorrectly addressed. This isn't timestamp support gone wrong, it's the `fromRow` failing to generate, but I am quite surprised you are still able to compile the code.

Could you please post your `case class CompanyWideGenderDistributionYearly` as well? I will look further into this.

**Comment by alexflav23** [comment]
Hi @mamdouhweb should be fixed in 2.24.2 I believe, making this primitive visible by default.
